x-pack/plugins/enterprise_search/public/applications/applications/components/playground/playground.tsx

```diff
@@ -39,9 +42,29 @@ export const Playground: React.FC = () => {
           }),
         ]}
         pageHeader={{
-          pageTitle: i18n.translate('xpack.enterpriseSearch.content.playground.headerTitle', {
-            defaultMessage: 'Playground',
-          }),
+          pageTitle: (
+            <EuiFlexGroup>
+              <EuiFlexItem grow={false}>
+                <span data-test-subj="chat-playground-home-page-title">
```

Comment:
Are you use the `<span>` here just for the `data-test-subj` attribute? if so you should be able to put that directly on the `<EuiFlexItem>`